packages/design-tokens/src/tokens/functional/components/card/colors.json

```diff
@@ -5,124 +5,6 @@
         "value": "var(--brand-color-canvas-default)",
         "dark": "var(--brand-color-canvas-subtle)"
       }
-    },
-    "icon": {
```

Comment:
We're missing a changeset in this PR, could we get one added please, and also capture in the Primitives one that we've removed these tokens?

Reply:
Good spot, thank you